webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionNameFormatter.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+
+export const subscriptionNameFormatter = (value, { rowData }) => {
+  let cellContent;
+
+  if (rowData.collapsible) {
+    cellContent = (
+      <span>
```

Comment:
Nitpick: why are these `<span>`s necessary?

Reply:
Good point - the tests aren't passing because of the inclusion of the tag

Reply:
@waldenraines They are not actually. Thanks.